docker/accumulo/entrypoint.sh

```diff
@@ -19,10 +19,12 @@ test -z "${ACCUMULO_INSTANCE_NAME}" && ACCUMULO_INSTANCE_NAME="accumulo"
 
 if [ "$1" = "accumulo" ] && [ "$2" = "master" ]; then
 	# Try to find desired root password from trace config
-	TRACE_USER=$(xmlstarlet sel -t -v "/configuration/property[name='trace.user']/value" ${ACCUMULO_CONF_DIR}/accumulo-site.xml)
-	if [ "${TRACE_USER}" = "root" ]; then
-		PASSWORD=$(xmlstarlet sel -t -v "/configuration/property[name='trace.token.property.password']/value" ${ACCUMULO_CONF_DIR}/accumulo-site.xml)
-	fi
+    if [ -f "${ACCUMULO_CONF_DIR}/accumulo-site.xml" ]; then
+        TRACE_USER=$(xmlstarlet sel -t -v "/configuration/property[name='trace.user']/value" ${ACCUMULO_CONF_DIR}/accumulo-site.xml)
+        if [ "${TRACE_USER}" = "root" ]; then
+            PASSWORD=$(xmlstarlet sel -t -v "/configuration/property[name='trace.token.property.password']/value" ${ACCUMULO_CONF_DIR}/accumulo-site.xml)
+        fi
+    fi
```

Comment:
Indenting wrong

Reply:
Is it? How would you recommend?

Reply:
The `if` should line up with the comment above it. Looks like it all needs to be shifted over to the right by one. This would lien it up with the comment and the indenting used elsewhere in this file.
Alternatively, the whole thing could be fixed as most of it is a bit too far indented.
This comment was more than the new bits added don't line up with what was there before.

Reply:
I'm sorry I still don't see what you mean, to me this looks right as it is a nested if. Could you send what you think it should be?

Reply:
This and the other similar comment have both been resolved by @ctas582